templates/layout.jade

```diff
@@ -31,3 +31,11 @@ html(lang='en')
             p &copy; #{ new Date().getFullYear() } #{ locals.owner } &mdash; powered by&nbsp;
               a(href='https://github.com/jnordberg/wintersmith') Wintersmith
               //- please leave the "powered by" if you use the design
+    //- Google analytics
+    script
+     | (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+     | (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+     | m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+     | })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+     | ga('create', 'UA-49880770-1', 'auto');
+     | ga('send', 'pageview');
```

Comment:
OR

- Create development Google Analytics account and point to different GA id's in dev/production (maybe via a config, maybe via env vars)

```jade
script
  | window.env = {gaId: #{process.env.google_analytics_id};
script.
  // ...
  ga('create', window.env.gaId, 'auto');
```